src/test/java/org/ow2/proactive/connector/iaas/fixtures/CredentialsFixtures.java

```diff
@@ -25,17 +25,30 @@
  */
 package org.ow2.proactive.connector.iaas.fixtures;
 
-import org.ow2.proactive.connector.iaas.model.Credentials;
+import org.ow2.proactive.connector.iaas.model.InfrastructureCredentials;
+import org.ow2.proactive.connector.iaas.model.InstanceCredentials;
 
 
 public class CredentialsFixtures {
 
-    public static Credentials getCredentials(String username, String password) {
-        return new Credentials(username, password, "");
+    public static InstanceCredentials getInstanceCredentials(String username, String password) {
+        return new InstanceCredentials(username, password, null, null);
```

Comment:
Is it not possible to use some Optional instead of `null` in order to prevent some NPE.